crud/select/filters.lua

```diff
@@ -44,7 +44,11 @@ local function get_index_fieldnos(index)
     local index_fieldnos = {}
 
     for _, part in ipairs(index.parts) do
-        table.insert(index_fieldnos, part.fieldno)
+        if part.path ~= nil then
+            table.insert(index_fieldnos, string.format("[%d]%s", part.fieldno, part.path))
+        else
+            table.insert(index_fieldnos, part.fieldno)
+        end
```

Comment:
I think this change should be covered with test https://github.com/tarantool/crud/blob/master/test/unit/select_filters_test.lua